fractal/response.py

```diff
@@ -35,8 +36,3 @@ def check_response(res, expected_status_code=200, coerce=False):
         logging.error(f"Server error message: {data}\n")
         logging.error("Terminating.\n")
         exit(1)
-
-    if coerce:
-        return coerce(**data)
-    else:
-        return data
```

Comment:
Is there any special reason for removing this line?

I think it's useful to always return the json content, so that we don't have to always repeat the `something = res.json()` statement in the other functions calling `check_response`.

Reply:
Fixed, thanks.